test/package/ancestry.nut.ts

```diff
@@ -0,0 +1,170 @@
+/*
```

Comment:
this is completely independent from the e2e nut so I split it out (can run parallel)